pkg/convert/convert.go

```diff
@@ -8,3 +10,15 @@ func convertToMessageFormatSingular(message string) string {
 
 	return "{" + message + "}"
 }
+
+// convertFromMessageFormatStrToStr function replaces '{' and '}' only if message starts with '{' and ends with '}'.
+func convertFromMessageFormatStrToStr(message string) string {
+	if strings.HasPrefix(message, "{") && strings.HasSuffix(message, "}") {
```

Comment:
The value stored in `Message` field is ALWAYS encoded using MessageFormat. Is this function temporary?